README.md

```diff
@@ -157,6 +158,11 @@ execute
   Placeholder for the next version (at the beginning of the line):
   ### **WORK IN PROGRESS**
 -->
+
+### 8.2.2 (2024-10-05)
+* (@rockflopp) Added Shelly 2 PM gen 3 Integration
```

Comment:
Don't add a version to the readme. Use the WORK IN PROGRESS heading

Reply:
Done